docs/source-app/levels/basic/hello_components/pl_multinode.py

```diff
@@ -5,8 +5,7 @@
 
 
 class LightningTrainerDistributed(L.LightningWork):
-    @staticmethod
-    def run():
+    def run(self):
```

Comment:
I was only able to run with the static method. Pretty sure this won't work

Reply:
> I was only able to run with the static method.
On latest release, yes. But on master this was recently changed so the example needs to be updated now to match